packages/react/src/hooks/useBreakpointValue.ts

```diff
@@ -2,31 +2,44 @@ import { Breakpoint } from '../primitives/types/responsive';
 import { getValueAtCurrentBreakpoint } from '../primitives/shared/responsive/utils';
 import { useBreakpoint } from '../primitives/shared/responsive/useBreakpoint';
 import { useTheme } from './useTheme';
-import { ThemeStylePropKey } from '../primitives/types/theme';
+import { getStyleValue } from '../primitives/shared/getStyleValue';
+import { isDesignToken } from '@aws-amplify/ui';
+
+interface UseBreakpointValue<T = unknown> {
+  (
+    values: Record<string, T> | T[],
+    defaultBreakpoint?: Breakpoint,
+    propKey?: string
+  ): T | string | number | null;
+}
 
 /**
  * [📖 Docs](https://ui.docs.amplify.aws/react/theming/responsive#usebreakpointvalue)
  */
-export function useBreakpointValue<T>(
-  values: Record<string, T> | T[],
-  defaultBreakpoint?: Breakpoint,
-  propKey?: string
-): T | string {
+export const useBreakpointValue: UseBreakpointValue = (
+  values,
+  defaultBreakpoint = 'base',
```

Comment:
huh, I thought this was already the default

Reply:
it was, but it was in the body of the function with `defaultBreakpoint ?? 'base'` I just made this change to clean it up a bit :)